utils/exceptions.py

```diff
@@ -15,28 +15,31 @@
 """
 
 class VFATDACBiasCannotBeReached(ValueError):
```

Comment:
Why does this class still exist? Isn't the spirit of this PR to merge all exceptions from DAC scan analysis into `DACAnalysisException`? From what I see, it is not raised anywhere anymore, but is caught in different places (where `DACAnalysisException` should be, e.g. in `ana_scans.py`).

Reply:
removed

Reply:
and replaced in ana_scans.py now

Reply:
`VFATDACBiasCannotBeReached` still present in `anaDACScan.py`.

Could you be more careful with your PR in the legacy branches? The policy is to not accept new features, particularly, for quality of life features. We made an exception for this PR which was already open, but one must be very cautious not to add any new bug.